### PR TITLE
GAWB-2638: Add Group Alert Message

### PR DESCRIPTION
**Pull request by rushtong**
Addresses: https://broadinstitute.atlassian.net/browse/GAWB-2638

Add a group alert message to:
1. The edit method permissions modal
2. The export method to workspace sync modal
3. The add user to workspace sync modal
4. The import method to workspace sync modal

---

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
1. I don't see the message inside the one-click-sharing dialogs
2. I realize you copied-and-pasted from the Jira ticket, but I think the message itself could use some wordsmithing for grammar.

**Comment by rushtong** [comment]
@davidangb Here is a screen shot of where I added it for one-click sharing. It's also in the method permissions modal. Did I miss another spot for this?
screen shot 2017-10-11 at 9 42 39 am

File: src/cljs/main/broadfcui/common/method/messages.cljs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+   ))
+
+
+(defn get-methods-repo-group-alert []
+  [:div {:style {:paddingBottom "1em" :fontSize "100%"}}
+   (icons/icon {:style {:fontSize 22 :color (:exception-state style/colors) :marginRight "1rem"}}
+               :warning)
+   (str "Currently user groups are not supported in the Method Repository. "
+        "That is, you cannot share a method/configuration with a user group. "
+        "We appreciate your patience as we work on developing this feature.")])
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/common/method/messages.cljs]: End file with newline

File: src/cljs/main/broadfcui/common/method/messages.cljs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+(ns broadfcui.common.method.messages
+  (:require
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.style :as style]
+   [broadfcui.utils :as utils]
+   ))
+
+
+(defn get-methods-repo-group-alert []
+  [:div {:style {:paddingBottom "1em" :fontSize "100%"}}
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/common/method/messages.cljs]: If you make this a `p` you probably won't need the padding; also, avoid `em` for `rem`

File: src/cljs/main/broadfcui/common/method/messages.cljs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+   (icons/icon {:style {:fontSize 22 :color (:exception-state style/colors) :marginRight "1rem"}}
+               :warning)
+   (str "Currently user groups are not supported in the Method Repository. "
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/common/method/messages.cljs]: You shouldn't have to `str` these.

**Comment by rushtong** [comment]
@davidangb, @zarsky-broad - please re-review post-merge.

File: src/cljs/main/broadfcui/page/workspace/summary/synchronize.cljs [hunk 4]
```diff
@@ -25,15 +26,18 @@
...
           (when unowned-methods
             [:div {}
              (if owned-methods
-               "Additionally, users will need to request access to the following methods:"
-               "In order to run the methods configured for this workspace, users will need to request
-                access to the following:")
+               [:div {}
+                (messages/get-methods-repo-group-alert)
+                "Additionally, users will need to request access to the following methods:"
+                "In order to run the methods configured for this workspace, users will need to request
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/page/workspace/summary/synchronize.cljs]: This `if` is broken now.

Comment (rushtong) [reply src/cljs/main/broadfcui/page/workspace/summary/synchronize.cljs]: omg it took me a half hour to see why I broke it. fixing...

**Comment by rushtong** [comment]
Email is out to PO for a request to change the message language.

File: src/cljs/main/broadfcui/common/method/messages.cljs [hunk 5]
```diff
@@ -0,0 +1,15 @@
+(ns broadfcui.common.method.messages
+  (:require
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.style :as style]
+   [broadfcui.utils :as utils]
+   ))
+
+
+(defn get-methods-repo-group-alert []
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/common/method/messages.cljs]: this should be a `def` rather than a `defn`, come to think of it

**Review by zarsky-broad — APPROVED** [review]

**Review by MattPutnam — APPROVED** [review]
👍

**Pull request merged**